layouts/shortcodes/note.html

```diff
@@ -1,3 +1,11 @@
-<div class="alert alert-info note callout" role="alert">
-  <strong>{{ T "note" }}</strong> {{ trim .Inner " \n" | markdownify }}
-</div>
\ No newline at end of file
+<!-- copied from Docsy alert shortcode -->
+{{ $_hugo_config := `{ "version": 1 }` }}
+{{ $color := "info" }}
+<div class="alert alert-{{ $color }}" role="alert">
+{{ with ( T "note" ) }}<h4 class="alert-heading">{{ . | safeHTML }}</h4>{{ end }}
+{{ if eq .Page.File.Ext "md" }}
+    {{ .Inner | markdownify }}
+{{ else }}
+    {{ .Inner | htmlUnescape | safeHTML }}
+{{ end }}
```

Comment:
The rendering issue incase of nested shortcodes [here](https://deploy-preview-46232--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) is caused by indentation within the shortcode. To resolve this, we can remove the indentation in the line where the content of the note is rendered, which should correct the rendering issue. Please refer below suggestion:

```suggestion
{{ if eq .Page.File.Ext "md" }}
{{ .Inner | markdownify }}
{{ else }}
{{ .Inner | htmlUnescape | safeHTML }}
{{ end }}
```

Alternatively, my preference leans towards using template action delimiters with hyphens `{{-` `-}}` 
 ([refer doc](https://gohugo.io/templates/introduction/#whitespace)) to prevent whitespace issues caused by indentation conflicts (please refer to my other suggestion in other files in the PR)